tests/lib/rules/no-legacy-test-waiters.js

```diff
@@ -0,0 +1,88 @@
+const rule = require('../../../lib/rules/no-legacy-test-waiters');
+
+const { ERROR_MESSAGE } = rule;
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('no-legacy-test-waiters', rule, {
+  valid: [
```

Comment:
I would prefer if all of these test cases could be much shorter. We don't need full working code for test cases. We just need the import statement and the line that will get flagged by the lint rule, so ~2 lines in each test case. It's much easier to work with concise test cases, and it makes it easier for us to add large number of test cases then.

Reply:
Agree. Will reduce sizes.